### PR TITLE
frontend: Fix undefined appId breaking group edit

### PR DESCRIPTION
**Pull request by yolossn**
## How to use

1. Click on an application.
2. Click on a group.
3. Editing the group should work.

### CODE REVIEW

**Review by illume — APPROVED** [review]
👍

File: frontend/src/components/Groups/GroupEditDialog/GroupEditDialog.tsx [hunk 1]
```diff
@@ -42,6 +43,8 @@ export default function GroupEditDialog(props: GroupEditDialogProps) {
   const [groupEditActiveTab, setGroupEditActiveTab] = React.useState(0);
   const { t } = useTranslation();
   const theme = useTheme();
+  const params = useParams<{ appID: string }>();
```
Comment (ashu8912) [inline frontend/src/components/Groups/GroupEditDialog/GroupEditDialog.tsx:46]: small nitpick, 
`const { appID } = useParams<{ appID: string }>();`

Comment (yolossn) [reply frontend/src/components/Groups/GroupEditDialog/GroupEditDialog.tsx:46]: When initiating edit from group page `useParams` returns an object with both `appID` and `groupID`.

Comment (ashu8912) [reply frontend/src/components/Groups/GroupEditDialog/GroupEditDialog.tsx:46]: yupp and this way we ask for the appID on the same line, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

**Pull request merged**